### ISSUE TITLE
problems on chinese dataset

### ISSUE THREAD

**Issue by yuzuda283** [ming024/FastSpeech2#17]
hi, i train fastspeech2 on the biaobei dataset. the result seems that the pitch predictor and the duration predictor doesn't work well, but if you input the groud truth, you can get a good result. so does the your demo use the ground truth pitch and energy or use the predictor to predict the pitch and energy?  i find other fastspeech2 code on tensorflowTTS and espnet2, they use the structure different from the orginal paper. they use length regulator after all the predictors finished, and pitch and energy predictor works well.

**Comment by guomin** [comment]
https://github.com/ming024/FastSpeech2/issues/1

**Comment by yuzuda283** [comment]
> 跟这个 #1 同一个问题
> 你找到解决办法了吗

看一下espnet 和 tensorflowTTS里的模型结构和训练方法， 和原论文的不一样。

**Comment by leijue222** [comment]
@yuzuda283 请问用Fastspeech2训练前，如何对标贝数据进行预处理？

**Comment by yuzuda283** [comment]
> @yuzuda283 请问用Fastspeech2训练前，如何对标贝数据进行预处理？

查tensorflowTTS 里面有他们处理的方法。

**Comment by leijue222** [comment]
> > @yuzuda283 请问用Fastspeech2训练前，如何对标贝数据进行预处理？
> 
> 查tensorflowTTS 里面有他们处理的方法。

谢谢，我按照tensorflowTTS提供的命令，对文件进行了处理。
`tensorflow-tts-preprocess --rootdir ./baker --outdir ./dump_baker --config preprocess/baker_preprocess.yaml --dataset baker

tensorflow-tts-normalize --rootdir ./dump_baker --outdir ./dump_baker --config preprocess/baker_preprocess.yaml --dataset baker`
![dataprocompar](https://user-images.githubusercontent.com/30276789/97274931-42242180-1870-11eb-80d1-d9400c91204b.png)
结果的形式有些不同，得改下train.py。另外TensorflowTTS这个预处理的方法应该用的不是MFA？

**Comment by yuzuda283** [comment]
> > > @yuzuda283 请问用Fastspeech2训练前，如何对标贝数据进行预处理？
> > 
> > 
> > 查tensorflowTTS 里面有他们处理的方法。
> 
> 谢谢，我按照tensorflowTTS提供的命令，对文件进行了处理。
> 
> `tensorflow-tts-preprocess --rootdir ./baker --outdir ./dump_baker --config preprocess/baker_preprocess.yaml --dataset baker`
> `tensorflow-tts-normalize --rootdir ./dump_baker --outdir ./dump_baker --config preprocess/baker_preprocess.yaml --dataset baker`
> 
> ![dataprocompar](https://user-images.githubusercontent.com/30276789/97274931-42242180-1870-11eb-80d1-d9400c91204b.png)
> 结果的形式有些不同，得改下train.py。另外TensorflowTTS这个预处理的方法应该用的不是MFA？

标贝数据集自带标记不需要MFA， 英语的数据集要MFA对齐

**Comment by ming024** [comment]
closed #17

**Comment by JunZhan2000** [comment]
@yuzuda283 请问要自己把标贝数据集的标记转换成TensorflowTTS需要的格式吗，数据集里的是.interval格式的文件，FastSpeech2需要.npy
